Enchanted/Application/EnchantedApp.swift

```diff
@@ -34,9 +34,12 @@ struct EnchantedApp: App {
                     NSWindow.allowsAutomaticWindowTabbing = false
                 }
 #endif
-        }.commands {
+        }
+#if os(macOS)
```

Comment:
Oh, I should have tried running iOS target before submitting https://github.com/AugustDev/enchanted/pull/79 to catch this error. Thanks for fixing 🙏🏼

<img width="735" alt="image" src="https://github.com/AugustDev/enchanted/assets/7823011/16ec6c28-e26c-41b9-9f15-a6dcc4667763">